src/resmom/linux/alps.c

```diff
@@ -3187,26 +3189,42 @@ inventory_to_vnodes(basil_response_t *brp)
 			default:
 				continue;
 		}
-		for (seg=node->segments; seg; seg=seg->next) {
-			char	vname[128];
 
-			snprintf(vname, sizeof(vname), "%s_%ld_%d",
-				mpphost, node->node_id, seg->ordinal);
-			vname[sizeof(vname)-1]='\0';
+		totcpus = 0;
+		totmem = 0;
```

Comment:
Okay.  They were just before the for loop, until I moved out the counting of accelerators...moving it back.